optimum/exporters/onnx/base.py

```diff
@@ -372,7 +372,7 @@ def __init__(
         if use_past != self.use_present_in_outputs:
             logger.warning(
```

Comment:
This warning is always raised, why is the warning necessary? cc @michaelbenayoun

Reply:
Basically, it is when `use_past` and `use_present_in_outputs` do not have the same value. But you are right the current way is not working all the time. Let me fix that on my end on a small PR.

Reply:
yes I understand that from the condition! But I'm not sure why there is the warning? In which case would the two be exclusive?

Reply:
Thanks!